.github/helm-docs.sh

```diff
@@ -2,6 +2,13 @@
 
 set -euxo pipefail
 
+# check if helm-docs is installed
+if ! command -v helm-docs &> /dev/null
+then
```

Comment:
```suggestion
if ! command -v helm-docs &> /dev/null; then
```

To make the linter happy